### PR TITLE
Update changelogs and statistics for Mender 3.3.2.

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

File: 10.Mender-server/docs.md [hunk 1]
```diff
@@ -2178,6 +2178,161 @@ New changes in integration since 3.3.0:
...
+* Upgrade useradm-enterprise to 1.18.2.
+* Upgrade useradm to 1.18.2.
+
+#### iot-manager (1.0.4)
+
+New changes in iot-manager since 1.0.3:
+
+##### Bug Fixes
+
+* wrong capitalization for `Type`
```
Comment (oleorhagen) [inline 10.Mender-server/docs.md:2273]: Doesn't make any sense to me :p

**Review by oleorhagen — APPROVED** [review]
Well, it happens. Good thing we rememberd now :smile_cat:

File: 10.Mender-server/docs.md (hunk 1 above)
Comment (kacf) [reply 10.Mender-server/docs.md:2273]: Is it a doc change..? The author will have to stand for this.

Comment (oleorhagen) [reply 10.Mender-server/docs.md:2273]: haha, yup!

**Pull request merged**